docs/running-on-kubernetes.md

```diff
@@ -106,6 +106,33 @@ The above mechanism using `kubectl proxy` can be used when we have authenticatio
 kubernetes-client library does not support. Authentication using X509 Client Certs and oauth tokens
 is currently supported.
 
+### Determining the Driver Base URI
+
+Kubernetes pods run with their own IP address space. If Spark is run in cluster mode, the driver pod may not be
```

Comment:
This could perhaps be more concise or clear. Would appreciate anyone's thoughts on how this could be better explained.